standard-deployment.cfg

```diff
@@ -39,9 +41,18 @@ instance-eggs -=
 zcml-additional-fragments +=
     <include package="${buildout:client-policy}" />
 
+zope-manager-group =
+zope-manager-configuration =
+    <product-config ftw.zopemaster>
+        manager_group cn=${zope-manager-group},ou=groups,dc=4teamwork,dc=ch
```

Comment:
Honestly, I wouldn't add another layer of substitution here - instead I'd make the `zope-manager-group` the full qualified DN.

Otherwise this adds a level of indirection that makes it harder than necessary to debug issues with this if you need to piece together the full DN from two different buildout files, in different repos nonetheless, just to find out _which group exactly_ is being specified.

It also would mean that it's impossible to customize a the zope manager group to one that's not in that OU. We might want to keep those groups in the customer-specific OUs though at some point.

It also shouldn't cause more work - the `zope-manager-group`, if customized, will always be copy pasted (or at least it should) - that's not something you want to be typing manually and potentially fat-fingering.

Reply:
as discussed we now provide the full DN in the variable.